life/modernized/13-chapter-13.adoc

```diff
@@ -19,9 +19,13 @@ among whom was a governor of that country, and one of the chief Lutheran priests
 It came upon me in this place to salute the Princess
 and Countess with this following epistle.
 
-A salutation to Elizabeth, Princess Palatine, and Anna Maria de Homes, Countess of Homes,
-at Herwerden in Germany.
+[.embedded-content-document.epislte]
```

Comment:
Looks like you spelled epistle wrong here (epislte), and the linter caught it. 🙂 While you're fixing that, maybe the word "salutation" should be capitalized in the blurb heading too (both here and in the original). A Salutation to Elizabeth... etc.